hs_err_pid12596.log

```diff
@@ -0,0 +1,206 @@
+#
```

Comment:
this files shouldn't be in the pull request
please remove them or add to .gitignore file

Reply:
ok, done